src/commonMain/kotlin/dev/zacsweers/fieldspottr/About.kt

```diff
@@ -56,7 +59,12 @@ fun About(modifier: Modifier = Modifier) {
       value =
         withContext(Dispatchers.IO) {
           val bytes = Res.readBytes("files/aboutlibraries.json")
-          Libs.Builder().withJson(bytes.decodeToString()).build()
+          Libs.Builder().withJson(bytes.decodeToString()).build().let { libs ->
+            libs.copy(
+              // https://github.com/mikepenz/AboutLibraries/issues/1228
+              libraries = libs.libraries.toPersistentList().mutate { it.distinctBy(Library::name) }
```

Comment:
@ZacSweers `it.distinctBy` returns a new list, but `mutate` expects you to mutate the given list, you want this instead:
```kotlin
libs.libraries.distinctBy(Library::name).toPersistentList()
```